perlmod/Fink/Services.pm

```diff
@@ -1427,7 +1443,11 @@ sub get_osx_vers {
 	my $darwin_osx = get_darwin_equiv();
 	$sw_vers =~ s/^(\d+\.\d+).*$/$1/;
 	if ($sw_vers != $darwin_osx) {
-		die "$sw_vers does not match the expected value of $darwin_osx. Please run `fink selfupdate` to download a newer version of fink";
+		if ($sw_vers == 11.6 && $darwin_osx == 11.5) {
+			# special case where it's OK to have a mismatch
```

Comment:
```suggestion
		if ($sw_vers == 11.6 && $darwin_osx == 11.5)  || ($sw_vers == 12.6 && $darwin_osx == 12.5)) {
			# special cases in Big Sur and Monterey where it's OK to have a mismatch
```
Additional case for current Monterey version (12.6); on Ventura 13.0 and 13.1 (so far) versions are matching.

Reply:
Thanks I'll look into that, I'm planing to start on the dpkg changed for the Apple Silicons soon.